src/commands/api/update.js

```diff
@@ -70,6 +51,40 @@ class UpdateAPICommand extends BaseCommand {
     if (flags.publish) await publish.run([apiPathWithVersion])
     if (flags.setdefault) await setDefault.run([apiPathWithVersion])
   }
+
+  async handleUpdate(owner, name, apiVersion, flags) {
+    await this.executeHttp({
+      execute: () => getApi([owner, name, apiVersion, 'settings', 'version']),
```

Comment:
Should this be `private`
```suggestion
      execute: () => getApi([owner, name, apiVersion, 'settings', 'private']),
```

Reply:
Very good catch @conor-ward, thank you. Changing it.